lib/hotkeys.ts

```diff
@@ -0,0 +1,29 @@
+const MODIFIERS = {
```

Comment:
I am not sure that we need aliases for so simple case like we have here. 

Maybe we can use just `altKey` in the hotkey string and remove this object (which is hard to minify)?

Reply:
done